dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -358,7 +359,7 @@ def report(self, remove_disabled_flag: bool = False) -> dict:
 
         return profile
 
-    def _load_hist_helper(self, data):
+    def _load_stats_helper(self, data):
```

Comment:
just a re-name?

Reply:
yes since the function does a lot more now.

Reply:
Should add a docstring though.